server/@types/viewModels/index.d.ts

```diff
@@ -64,9 +65,9 @@ declare module 'viewModels' {
   /**
    * Collates the Additional Learning Needs (ALN) and Learning Difficulties and Disabilities (LDD) assessments that have been recorded in Curious
    */
-  export interface PrisonerSupportNeeds {
```

Comment:
`PrisonerSupportNeeds` was the original name for this type when
a) the LWP screen was called Support Needs
b) it only contained LDD assessments
c) ALN assessments were not a thing (this was written long before Curious 2 introduced ALN assessments)

It was a bad name because the data never described a "support need". It described assessments, which is not a "need" in itself